src/acitons/index.js

```diff
@@ -0,0 +1,9 @@
+export const addBook = book => ({
+  type: 'CREATE_BOOK',
+  payload: book,
```

Comment:
small typo error...... you names folder this component is as `acitons` instead of `actions`